Orange/widgets/utils/plot/owplotgui.py

```diff
@@ -575,14 +575,15 @@ def _get_callback(self, name, master=None):
         else:
             return name
 
-    def _check_box(self, widget, value, label, cb_name):
+    def _check_box(self, widget, value, label, cb_name, state):
```

Comment:
> I am not entirely satisfied with the solution for owplotgui. Could you check and propose what could be done better?

owplotgui should be abolished anyway, so I don't care much. (Idea was to unify GUIs of visualizations. Projections are now unified under a common base widget, and on the first glance, owplotgui is used exclusively in that widget (and derived ones). And even they use just small pieces, while most of it is unused. Check box for animating visualizations!?)

Still, please:

- rename the argument to `stateWhenDisabled`, otherwise this looks like initial state. Plus, it will be clearer that it is passed to `gui.checkBox`.
- give it a default value (`None`).

Reply:
Fixed. I hope appropriately.